examples/example-js.js

```diff
@@ -2,10 +2,11 @@ const { ClusterScanner, NonceScanner } = require('ssv-scanner');
 
 async function main() {
   const params = {
+    network: '',
     nodeUrl: '',
-    contractAddress: '',
     ownerAddress: '',
-    operatorIds: [],
+    contractAddress: '',
```

Comment:
It looks like --network param already knows what contract to be used

Reply:
The example doesn't work without it, possibly some kind of bug in the code. looking into it.

Reply:
@meshin-dev fixed. Let's discuss offline.